package.json

```diff
@@ -782,6 +788,27 @@
                         "type": "boolean",
                         "default": false,
                         "description": "%azureApiManagement.advancedPolicyAuthoringExperience%"
+                    },
+                    "azureApiManagement.selectedSubscriptions": {
+                        "type": "array",
+                        "description": "Selected Azure subscriptions",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "azureApiManagement.selectedTenant": {
+                        "type": "object",
+                        "description": "A specific tenant to sign in to",
```

Comment:
also use parameters to define strings of descriptions?

Reply:
thanks, updated.